.env.example

```
@@ -1,20 +1,21 @@
 GUILD_ID=
+
 GCP_CLIENT_EMAIL=
 GCP_PRIVATE_KEY=
 SPREADSHEET_ID=
-OPENAI_ORGID=
-OPENAI_PROJECTID=
-OPENAI_ASSISTANT_ID=
-PORT=
+
 LOG_FLARE_SOURCE=
 LOG_FLARE_API=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LOG_FLARE_API key should go before the LOG_FLARE_SOURCE key

(UnorderedKey)